topics/transcriptomics/tutorials/scrna-scanpy-pbmc3k/tutorial.md

```diff
@@ -871,7 +871,7 @@ Here we perform the PCA on the log-normalized expression values and compute the
 
 > ### {% icon hands_on %} Hands-on: Perform the PCA
 >
-> 1. **Cluster, infer trajectories and embed** {% icon tool %} with the following parameters:
+> 1. **Cluster,** infer trajectories and embed with scanpy {% icon tool %} with the following parameters:
```

Comment:
The tool name was updated to have the larger title.

See:
https://github.com/galaxyproject/tools-iuc/blob/9d7adf6b6dd22f25379336d2ab94bba513d9ad89/tools/scanpy/cluster_reduce_dimension.xml

Reply:
...but on https://usegalaxy.eu/ , there is still only "cluster" bold, isn't-it?

Reply:
...I see, now it has changed. I will modify my pull request accordingly

Reply:
Yeah I think the tools weren't refreshed earlier -- thanks for this